Create the accodion component and replace the main pages' content (#682)
rsksmart/devportal: rif/multisig/index.md

---
layout: rsk
title: RIF Savings and Vault Solution
description: Overview of the RIF Savings and Vault solution

> tags: rif gnosis multisig

Reviewer: ```suggestion
tags: rif, gnosis, multisig
```

---

> The objective of this project is to provide the RSK community with a safe and simple (from a developer's point of view) solution to create accounts that require multiple signatures to transact their funds.

Reviewer: ```suggestion
The objective of this project is to provide the RSK community with a safe and simple (from a technical POV) solution to create accounts that require multiple signatures to transact their funds.
```

## Abstract

The solution consists of four main modules:

* A suite of smart contracts that allow creating multi-sig accounts
* A Node.js and Web SDK for creating and operating multi-sig
* Proper documentation that will help developers to integrate the solution to different projects
* A sample application exposing a correct user experience for operating multi-sig.

<div class="container the-stack">
  <div class="row rif_blue_text">
    <div class="col">
      <div class="rns-index-box">
        <a href="get-started">Get Started</a>
        <br />
        <br />
        <p>Use the Javascript SDKs to create and operate multisig wallets</p>
      </div>
    </div>
    <div class="col">
      <div class="rns-index-box">
        <a href="https://sample-app-multisig.rifos.org/" rel="noopener" >Sample Application</a>
        <br />
        <br />

>         <p>Take a look at the the sample application, you will find handy snippets in the code</p>

Reviewer: ```suggestion
        <p>Take a look at the sample application, you will find handy snippets in the code</p>
```
